projects/sport-data-valley/manual_seeds/rheuma_responses.rb

```diff
@@ -0,0 +1,90 @@
+# frozen_string_literal: true
+
+require_relative 'helpers/people_helper'
+
+srand(123)
+
+START_DATE = Time.zone.parse('2021-11-24').beginning_of_day
+END_DATE = START_DATE + 3.weeks
+N_PARTICIPANTS = 5
+
+def range_0_10
+  rand(0.0..10.0).round(1)
+end
+
+# Returns a value in between 0 and 24 in increments of 0.5
+def n_hours
+  hours = rand(24*2 + 1)
+  hours * 0.5
+end
+
+def create_rheuma_responses(person, date)
+  return if rand > 0.9
+
+  protocol_name = 'daily_protocol_rheumatism'
+  puts "Creating responses for #{protocol_name}"
+  protocol = Protocol.find_by(name: protocol_name)
+  raise "Error: protocol named #{protocol_name} not found" unless protocol
+
+  protsub = ProtocolSubscription.find_by(person: person, protocol: protocol)
+  protsub ||= ProtocolSubscription.new(person: person, protocol: protocol)
+
+  protsub.state = ProtocolSubscription::COMPLETED_STATE
+  protsub.start_date = Time.new(2016).in_time_zone
+  protsub.end_date = Time.new(2017).in_time_zone
+  protsub.save!
+
+  questionnaire_name = 'Dagelijkse vragenlijst reuma'
```

Comment:
So in my opinion, a questionnaire name can always change, but its key stays the same always.

I know that we violate this assumption in some SDV seeds. And for this reason both name and key are unique, so in practice it doesn't matter which one you use to look up.

But maybe you can change this lookup to be by key instead (`daily_questionnaire_rheumatism`).